src/App.tsx

```diff
@@ -10,33 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [{ id, title }, setselectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    setselectedTab(tab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {title}
```

Comment:
use interpolation, it will be more easy to read  

`'Selected tab is ${title}'`